src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/prison/api/OtherPrisonerDetails.kt

```diff
@@ -6,4 +6,7 @@ import io.swagger.v3.oas.annotations.media.Schema
 class OtherPrisonerDetails(
   @Schema(description = "The prisoner number", example = "G0135GA", required = true)
   val prisonerNumber: String,
+
+  @Schema(description = "The prison code", example = "MDI", required = false)
+  val prisonId: String? = null,
```

Comment:
I would do non optional

Reply:
should I remove this check for now. will add it once confirmed

Reply:
removed the check for now. Will push this change through, do some more checks and then push this through.